### DISCUSSION TITLE
Voice Chat: Is this working?

### DISCUSSION THREAD

**Discussion by digidigital** [PadWorld-Entertainment/worldofpadman-fork#138] [Q&A]
Is the Voice Chat feature working? So far I have never heard someone online and never got a reaction when I tried to use this feature... 

I don't want to open a bug if it's just me having an issue on my side :)

**Comment by kai-li-wop** [comment]
I totally don't know. I know it used to work in WoP 1.2 but since WoP 1.5 I have the feeling, that it is not working correctly. Maybe @mgerhardy could have a look to check what needs to be done to enable it. I think voip is enabled on the beta server, at least it is set to "opus", what ever that means. Sorry for replying late here, haven't recognised it here.

**Reply by mgerhardy** [reply]
It should work fine - have you checked the voip-readme? especially `cl_voipSendTarget` might be of interest for you. Try to set this cvar to `all`. The default (`spatial`) will only send to people that are close to you.

Note to myself: We should really expose this in the ui and add proper tooltips

**Reply by mgerhardy** [reply]
see https://github.com/PadWorld-Entertainment/worldofpadman/pull/163

**Comment by ghost** [comment]
Is this really fully implemented?
I had a quick look at the code and I see a few bits that makes me wonder, for example 'trap_GetVoipTimes/CL_GetVoipTimes' isn't used/supported I think.
Is this the 'Tremulous style' of the VOIP implementation? If so than the 'team' support is missing.
